scripts/ci/check-cli-docs.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+"""Check if CLI documentation is up-to-date."""
+
+from __future__ import annotations
+
+import argparse
+import difflib
+import subprocess
+import sys
+from pathlib import Path
+
+# TODO(grtlr): Is there a better way to call this?
+default_command = ["cargo", "run", "--features", "release", "--", "man"]
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Check if CLI documentation is up-to-date")
+    parser.add_argument(
```

Comment:
Why is this an argument?

Reply:
In CI, I want to run against the actual binary that was built to avoid any additional work: https://github.com/rerun-io/rerun/pull/10864/files#diff-63182be3cf3f909cddc762c938835d1a29873b1f35b30f375fe2d0c9a7fb8abbR191